libfreerdp/codec/h264_mf.c

```diff
@@ -182,7 +182,7 @@ static HRESULT mf_create_output_sample(H264_CONTEXT_MF* sys)
 
 	if (FAILED(hr))
 	{
-		WLog_ERR(TAG, "MFCreateSample failure: 0x%08"PRIX32"", hr);
+		WLog_Print(h264->log, WLOG_ERROR, "MFCreateSample failure: 0x%08"PRIX32"", hr);
```

Comment:
looks like the context is named sys instead of h264 (that's why the VS builds fail)